images/autoperibolossync/Dockerfile

```diff
@@ -1,10 +1,10 @@
-FROM quay.io/centos/centos:stream9
+FROM registry.access.redhat.com/ubi9/ubi-minimal:latest
 
-ADD private-org-peribolos-sync /usr/bin/private-org-peribolos-sync
-ADD autoperibolossync /usr/bin/autoperibolossync
+#ADD private-org-peribolos-sync /usr/bin/private-org-peribolos-sync
```

Comment:
Why have these two lines been commented?

Reply:
These were for testing the image, let me fix this.